lib/puppet/parser/functions/contain.rb

```diff
@@ -25,6 +25,6 @@
 "
 ) do |classes|
   # Call the 4.x version of this function in case 3.x ruby code uses this function
-  Puppet.warn_once(:deprecation, '3xfunction#contain', "Calling function_contain via the Scope class is deprecated. Use Scope#call_function instead")
+  Puppet.warn_once(:deprecation, '3xfunction#contain', _("Calling function_contain via the Scope class is deprecated. Use Scope#call_function instead"))
```

Comment:
`function_contain` `Scope` and `Scope#call_function` are all keywords, should they be left out of translation?